### ISSUE TITLE
Guild Is Full cannot invite new people

### ISSUE THREAD

**Issue by Meliannia** [WarEmu/WarBugs#19794]
guild invite was working fine as paint last week but now we cant invite anyone
we even removed 15 inactive lowbies to free up space and still wont let us invite
Guild is <Ethics>

scrolling thru this website i found out the guild member limits 5 years ago was 80 but there's guilds nowadays with 1200 in now eg Bitterstone Thunderers so my guild cant have reached its cap as Bitterstone has 1200 members
has a new cap been introduced and no-one knows or is this just a temp invite bug or a roster bug?

**Comment by kpihuss** [comment]
Same problem in Tercio de Estalia, we can not invite

**Comment by lilimx** [comment]
Hey,

Guess it will be written in the next Patch notes.
on Monday 7th February we limited the max. number of members to 500.
So every guild with more than 500 members can't invite more players (That's why the error).
We did this, because the higher the number of members, the higher the risk that people stuck during the loading screen.
Like on live, the number of max. members were 500, we already lifted the amount up to 1500 people, but then this issue appeared. 
We are still trying to save the dwarfs from BT, because almost no one can login. So Max is on it to find a proper solution. The next tweak will be on Wednesday.

Sorry for the inconvenience
